test/api-digester/source-stability.swift.expected

```diff
@@ -7,6 +7,9 @@ Protocol IndexableBase has been removed (deprecated)
 Protocol MutableIndexable has been removed (deprecated)
 Protocol RandomAccessIndexable has been removed (deprecated)
 Protocol RangeReplaceableIndexable has been removed (deprecated)
+Func Array.append(contentsOf:) has been removed
+Func ArraySlice.append(contentsOf:) has been removed
+Func ContiguousArray.append(contentsOf:) has been removed
```

Comment:
oh, that's right. rdar://29074045. @airspeedswift Do we need migration help for these removal? If yes, could you comment by using `//` in a line before these removed decls?